scripts/install_ext.sh

```
@@ -0,0 +1,73 @@
+if [ ! -d "$VENV_DIR" ]; then
+    echo "Creating virtual environment..."
+    python3 -m venv "$VENV_DIR"
+fi
+
+echo "Activating virtual environment..."
+source "$VENV_DIR/bin/activate"
+
+# Check if nvm is working
+if ! command -v nvm >/dev/null 2>&1; then
+  echo "Working nvm installation not found. Installing nvm..."
+  export NVM_DIR="$([ -z "${XDG_CONFIG_HOME-}" ] && printf %s "${HOME}/.nvm" || printf %s "${XDG_CONFIG_HOME}/nvm")"
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)